RIS.js

```diff
@@ -2677,12 +2748,17 @@ var testCases = [
 				"seriesTitle": "Series Title",
 				"shortTitle": "Short Title",
 				"url": "URL",
-				"version": "Version",
```

Comment:
Hm... What was the status with this variable/name?

Reply:
It should be `versionNumber`.

Reply:
We tried to change that some time ago, but then there were some problems such that we had to revert the changes again, cf. https://github.com/zotero/translators/pull/1085 . However, I forgot the details...